iogt/templates/comments/form.html

```diff
@@ -3,11 +3,9 @@
 <form action="{% url 'molo.commenting:molo-comments-post' %}#comments-list" method="post" class="comment__form comment__form{{self.get_parent_section.get_effective_extra_style_hints}}">
 {% csrf_token %}
     {% if form.errors %}
-      <h6 class="comment__text">
+      <h6 class="comment__text {% if form.errors|length > 1 %}comment__text--plural{% endif %}">
         {% blocktrans count counter=form.errors|length %}
         Please correct the error below
-      </h6>
-      </h6 class="comment__text comment__text--plural">
```

Comment:
@Mitso Just noted this here, the `<h6>` tag should be around the `{% blocktrans %}` tag otherwise when we generate the PO file it will add these lines in the PO file to be translated too. In this case, we can do something like line 6